backbone.js

```diff
@@ -346,11 +353,13 @@
       changing        = this._changing;
       this._changing  = true;
 
+      current = this.attributes, prev = this._previousAttributes;
+
       if (!changing) {
-        this._previousAttributes = _.clone(this.attributes);
+        prev = this._previousAttributes = {};
+        for( attr in current ){ prev[ attr ] = current[ attr ]; }
```

Comment:
Not a big fan of this without evidence. This is essentially what `clone` does anyway (but it uses an extra loop). This will also not support some of the iteration bugs for things like `hasOwnProperty`, etc. that we've fixed in underscore